src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -35,12 +37,13 @@ export class ChartReleasesComponent implements OnInit {
   private refreshTable: Subscription;
   private refreshForm: Subscription;
   
-  public emptyPageConf = {
+  public emptyPageConf: EmptyConfig = {
+    type: EmptyType.loading,
+    large: true,
     title: helptext.message.loading,
```

Comment:
Lets add a message too.

"Applications you install will automatically appear here. Click below and browse the TrueNAS catalog to get started."